wherehows-dao/src/main/java/wherehows/models/JiraTicket.java

```diff
@@ -13,20 +13,59 @@
  */
 package wherehows.models;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+import lombok.Data;
+
+
+@Data
+@Entity
+@Table(name = "dir_external_user_info")
```

Comment:
I had a discussion with Yi yesterday,  noArgConstructor is still needed, @DaTa won't generate one for you.
Hibernate needs noArgConstructor to generate the class using reflection.